learning.py

```diff
@@ -655,7 +655,7 @@ def predict(example):
 
 
 def NeuralNetLearner(dataset, hidden_layer_sizes=[3],
-                     learning_rate=0.01, epochs=100, activation = sigmoid,momentum=False):
+                     learning_rate=0.01, epochs=100, activation = sigmoid, momentum=False):
```

Comment:
Not your commit, but can you also remove spaces from around the equal sign in 'activation = sigmoid'?

Reply:
ya sure!